compiler/src/iree/compiler/Codegen/LLVMGPU/ConvertToROCDL.cpp

```diff
@@ -325,8 +325,9 @@ struct ConvertToROCDLPass final
       populateVectorToLLVMConversionPatterns(converter, llvmPatterns);
       vector::populateVectorTransferLoweringPatterns(llvmPatterns,
                                                      /*maxTransferRank=*/1);
-      populateGpuToROCDLConversionPatterns(converter, llvmPatterns,
-                                           gpu::amd::Runtime::Unknown);
+      populateGpuToROCDLConversionPatterns(
+          converter, llvmPatterns, gpu::amd::Runtime::Unknown,
+          maybeChipset.value_or(amdgpu::Chipset()));
```

Comment:
I think we should materialize the chipset value earlier on since the same pattern appears a few lines above

Reply:
done! :)